samples/boards/nrf52/mesh/onoff_level_app/README.rst

```diff
@@ -0,0 +1,101 @@
+.. _bluetooth-mesh-onoff-sample:
+
+Bluetooth: Mesh OnOff & Level Model
+###########################
+
+Overview
+********
+
+This is a simple application demonstrating a Bluetooth mesh node.
+
+Root element has Generic OnOff Server, Generic OnOff Client, Generic Level Server, Generic Level Client models.
+
+Prior to provisioning, an unprovisioned beacon is broadcast that contains
+a unique UUID. It is obtained from the device address set by Nordic in the
+FICR. Each button controls the state of its
```

Comment:
for clarity, add:

   in the Factory information configuration registers (FICR).